BFE_RShiny/flamingo/R/ViewFilesInTable_module.R

```diff
@@ -307,37 +309,58 @@ ViewFilesInTable <- function(input, output, session,
   })
 
   # Exposure table
+  output$FVExposureSelected <- renderUI({
+    extension <-  strsplit(result$currentFile, split = "\\.") %>% unlist() %>% tail(n = 1)
+    if (extension == "csv") {
+      DTOutput(ns("dt_FVExposureSelected"))
+    } else if (extension == "json") {
+      verbatimTextOutput(ns("json_FVExposureSelected"))
+    } else {
+      textOutput(ns("text_FVExposureSelected"))
+    }
+  })
+
   output$dt_FVExposureSelected <- renderDT(
-    if (!is.null(result$tbl_fileData) && nrow(result$tbl_fileData) > 0 ) {
+    if (!is.null(result$tbl_fileData) && nrow(result$tbl_fileData) > 0) {
```

Comment:
```suggestion
    if (!is.null(result$tbl_fileData)) {
```
The json obj does not have nrow